README.md

```diff
@@ -50,31 +50,25 @@ Update `conda`
 conda update conda
 ```
 
-Add the `mosdef`, `conda-forge`, and `omnia` channels
-for `conda` to search through when installing packages.
+Add the `conda-forge` channels for `conda` to search through when installing packages.
 
 ```bash
-conda config --add channels mosdef
 conda config --add channels conda-forge
-conda config --add channels omnia
 ```
 
-Create a new python 3.6 development environment named `simulation36` 
+Create a new python  environment named `cybercamp2022` 
 that includes many of
 the packages needed to build systems, run simulations, and
 analyze the data.
 
 ```
-conda create -n simulation36 python=3.6 mbuild foyer hoomd matplotlib signac signac-flow fresnel gsd freud jupyter py3dmol
+conda create -n cybercamp2022 mbuild foyer hoomd matplotlib signac signac-flow fresnel openbabel jupyter jupyterlab
```

Comment:
py3dmol still isn't a hard dependency for mBuild, so we should have them install that here.